src/thunk.js

```diff
@@ -0,0 +1,26 @@
+var compose = require('./compose');
+var nAry = require('./nAry');
+var partial = require('./partial');
+
+/**
+ * Accepts as its arguments a function and any number of values and returns a function that,
+ * when invoked, calls the original function with only the values originally provided.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig (a -> b -> ... -> i -> j -> ... -> m -> n) -> a -> b -> ... -> i -> (j -> ... -> m -> n)
+ * @param {Function} fn The function to invoke.
+ * @param {...*} [args] Arguments to invoke `fn` with.
+ * @return {Function} A new function wrapping `fn`. When invoked, it will call `fn`
+ *         with `args`, while disregarding any new arguments.
+ * @see R.partial
+ * @example
+ *
+ *      var add = (a, b) => a + b;
+ *      var sum = (...args) => args.reduce(add, 0);
+ *
+ *      var sumThunk = R.thunk(sum, 1, 2)
+ *      sumThunk(5); //=> 3 // 5 is disregarded
```

Comment:
While it is good to demonstrate extra arguments are ignored, I think this example should also include the more common use case of evaluating the thunk with no args.

Perhaps including two examples of the same thunk being called could be more informative.
```js
var sumThunk = R.thunk(sum, 1, 2)
sumThunk();  //=> 3
sumThunk(5); //=> 3 // 5 is disregarded
```